vmdb/db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Methods.class/__methods__/postprovision.rb

```diff
@@ -0,0 +1,14 @@
+#
+# Description: This method examines the orchestration stack provisioned
+#
+$evm.log("info", "Starting Orchestration Post-Provisioning")
+
+# In the following example all the outputs from the stack are dumped.
+# You can add logic to process the stack object in VMDB
+service = $evm.root["service_template_provision_task"].destination
+stack = $evm.vmdb('orchestration_stack').find_by_ems_ref(service.stack_ems_ref)
+
+$evm.log("info", "Outputs from stack #{stack.name}")
+stack.outputs.each do |output|
+  $evm.log("info", "Key #{output.key}, value #{output.value}")
```

Comment:
@bzwei Is there potential to log sensitive data here like passwords?  I think this is fine to have as an example but I would not have it run by default.  I would suggest making this logic a method and then comment out just the call to the method.